examples/disagg_prefill.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
```

Comment:
the usage is quite experimental, right? I suppose they are prone to change, and we should add follow-up PRs to add `vllm connect` .

I suggest adding comment and warning here, saying that this feature is experimental, and the usage is subject to change.

Reply:
Yes ur right. Just updated the script to warn people that the usage is experimental.